.github/workflows/integration-tests.yml

```
@@ -0,0 +1,164 @@
+                        - 'sdk/**'
+                        - 'examples/**'
+                        - '.github/workflows/integration-tests.yml'
+
+jobs:
+        android-app:
+                name: Test Android Example App
+                runs-on: ubuntu-latest
+
+                steps:
+                - name: Checkout code
+                        uses: actions/checkout@v4
```

actionlint (1.7.9)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)